pyanaconda/modules/subscription/initialization.py

```diff
@@ -33,12 +35,26 @@
 from pyanaconda.anaconda_loggers import get_module_logger
 log = get_module_logger(__name__)
 
+RHSM_SYSTEMD_UNIT_NAME = "rhsm.service"
+
+
+def check_initial_conditions():
+    """Can the Subscription service run?"""
+
+    # Exclude the dir and image installations.
+    if not conf.target.is_hardware:
+        log.debug("subscription: Unsupported type of the installation target. Quit.")
```

Comment:
Suggestion: "subscription: Unsupported type of the installation target. Subscription module will not be started."

Reply:
Fixed.